plugins/modules/docker_plugin.py

```diff
@@ -0,0 +1,343 @@
+                        self.existing_plugin.configure(prepare_options(self.parameters.plugin_options))
+                    except APIError as e:
+                        self.client.fail(text_type(e))
+                    self.results['actions'].append("Updated plugin %s settings" % self.parameters.plugin_name)
+                    self.results['changed'] = True
+            else:
+                self.fail("Cannot update the plugin: Plugin does not exist")
+
+    def present(self):
+        differences = DifferenceTracker()
+        if self.existing_plugin:
+            differences = self.has_different_config()
+
+        self.diff_tracker.add('exists', parameter=True, active=self.existing_plugin is not None)
+
+        if self.existing_plugin:
+            self.update_plugin()
+        else:
+            self.install_plugin()
+
+        if self.diff or self.check_mode or self.parameters.debug:
+            self.diff_result['differences'] = differences.get_legacy_docker_diffs()
+            self.diff_tracker.merge(differences)
+
+        if not self.check_mode and not self.parameters.debug:
+            self.results.pop('actions')
+
+    def absent(self):
+        self.remove_plugin()
+
+    def enable(self):
+        timeout = self.parameters.enable_timeout
+        if self.existing_plugin:
+            if not self.check_mode and not self.existing_plugin.enabled:
+                try:
+                    self.existing_plugin.enable(timeout)
+                except APIError as e:
+                    self.client.fail(text_type(e))
+                self.results['actions'].append("Enabled plugin %s" % self.parameters.plugin_name)
+                self.results['changed'] = True
+        else:
+            self.install_plugin()
+            try:
+                self.existing_plugin.enable(timeout)
+            except APIError as e:
+                self.client.fail(text_type(e))
+            self.results['actions'].append("Enabled plugin %s" % self.parameters.plugin_name)
+            self.results['changed'] = True
+
+    def disable(self):
+        if self.existing_plugin:
+            if not self.check_mode and self.existing_plugin.enabled:
+                try:
+                    self.existing_plugin.disable()
+                except APIError as e:
+                    self.client.fail(text_type(e))
+                self.results['actions'].append("Disable plugin %s" % self.parameters.plugin_name)
+                self.results['changed'] = True
+        else:
+            self.fail("Plugin not found: Plugin does not exist.")
+
+
+def main():
+    argument_spec = dict(
+        plugin_name=dict(type='str', required=True),
+        state=dict(type='str', default='present', choices=['present', 'absent', 'enable', 'disable']),
+        plugin_options=dict(type='dict', default={}),
+        debug=dict(type='bool', default=False),
+        force_remove=dict(type='bool', default=False),
+        enable_timeout=dict(type='int', default=0)
```

Comment:
```suggestion
        enable_timeout=dict(type='int', default=0),
```
In general it's a good idea to have a trailing comma after the last parameter in such situations. When a new one is added and the comma is here, this line does not need to be changed and the diff will be smaller.